helpers/reminderHelper.js

```diff
@@ -0,0 +1,26 @@
+const { getGuild, saveGuild } = require('./guildData')
+const { discordTime } = require('./DateTimeHelper')
+const { foemp } = require('./foemp')
+
+async function setReminder (interaction) {
+  const guild = await getGuild(interaction.guildId)
+  const currentTime = discordTime().toEpochSecond().toString()
+  const reminderMinutes = interaction.fields.getTextInputValue('reminderTime')
+  if (isNaN(reminderMinutes)) {
+    return await interaction.reply({ content: `Je hebt het aantal minuten ingesteld (${reminderMinutes}) in een formaat dat ik niet begrijp. Probeer een integer (bv. 10), ${foemp()}!`, ephemeral: true })
+  }
+  const timeToBeReminded = discordTime().plusMinutes(interaction.fields.getTextInputValue('reminderTime')).toEpochSecond()
+  const message = interaction.fields.getTextInputValue('reminderMessage')
+  const memberId = interaction.member.id
+  guild.reminders[currentTime] = { memberId, timeToBeReminded, message }
+  await saveGuild(guild)
+  if (reminderMinutes === '1') {
+    await interaction.reply({ content: `Reminder ingesteld voor binnen ${reminderMinutes} minuut!` })
```

Comment:
You can simply put '1' in the string, as no other value will be able to reach this line